data/json/monsterdrops/zombie_military_pilot.json

```diff
@@ -6,13 +6,13 @@
     "magazine": 100,
     "ammo": 60,
     "entries": [
-      { "group": "clothing_military_pilot", "prob": 65, "damage-min": 1, "damage-max": 4 },
-      { "group": "gear_soldier_sidearm", "prob": 50, "damage-min": 1, "damage-max": 4 },
-      { "item": "two_way_radio", "prob": 50, "damage-min": 1, "damage-max": 4 },
+      { "group": "clothing_military_pilot", "damage": [ 1, 4 ] },
+      { "item": "holster", "contents-group": "military_standard_pistols" },
```

Comment:
A _guranteed_ pistol/holster drop? That doesn't seem likely

Reply:
American air crews carry pistols. Sometimes it will be in something other than a holster but I don't think that's a significant detail to model.